configs/maventa.json

```diff
@@ -1,22 +1,32 @@
 {
   "index_name": "maventa",
   "start_urls": [
-    "https://documentation.maventa.com/"
+    "https://documentation.maventa.com/",
+    {
+      "url": "https://documentation.maventa.com/overview/",
+      "selectors_key": "overview"
+    }
   ],
   "stop_urls": [
     "//.*//"
   ],
   "selectors": {
-    "lvl0": ".col-sm-9 h1",
-    "lvl1": ".col-sm-9 h2",
-    "lvl2": ".col-sm-9 h3",
-    "lvl3": ".col-sm-9 h4",
-    "lvl4": ".col-sm-9 h5, .col-sm-9 summary",
-    "lvl5": ".col-sm-9 h6",
-    "text": ".col-sm-9 p, .col-sm-9 li"
+    "default": {
+      "lvl0": ".col-sm-9 h1",
+      "lvl1": ".col-sm-9 h2",
+      "lvl2": ".col-sm-9 h3",
+      "lvl3": ".col-sm-9 h4",
+      "lvl4": ".col-sm-9 h5, .col-sm-9 summary",
+      "lvl5": ".col-sm-9 h6",
+      "text": ".col-sm-9 p, .col-sm-9 li"
+    },
+    "overview": {
+      "lvl0": ".col-sm-9 h1",
+      "lvl1": ".col-sm-9 h2"
```

Comment:
You need to define all selectors within a set. Please add other lvls

Reply:
page contains only h1 and h2 and text. text added